doc/users/next_whats_new/pie_hatch.rst

```diff
@@ -6,6 +6,7 @@ a hatch or list of hatches:
 
 .. plot::
     :include-source: true
+    :alt: Two pie charts with 3 identically colored and sized wedges. Each wedge in the first chart is a distinct pattern from the list passed to hatch. Every wedge in the second chart has the same string pattern passed to hatch.
```

Comment:
```suggestion
    :alt: Two pie charts with three identically colored and sized wedges. Each wedge in the first chart has a distinct pattern from the list passed to hatch. Every wedge in the second chart has the same string pattern passed to hatch.
```
? (If nothing else for consistency with "Two".)

Reply:
I'm finding some of the `alt` text a confusing mix of what was done and what is shown.  I assume people who need this text can read the code for what was done.  I'd suggest that the alt text should just explain as clearly as possible what is visually shown.  Have we delineated somewhere what the goal of alt-text is for examples?  For the above I'd do:

``` Two pie charts, with a small blue slice, a medium orange slice, and a large green slice. The first pie has a dot hatching for the blue slice, a small open circle hatch for the orange, and a large open circle for the green.  The second has the same large open circle with a dot hatch for each slice.
```

Reply:
And by "some" I mean a lot of the text in the open PR about this.

Reply:
> So I go back and forth on how much semantic info to add b/c the guidance I've read is that it's less about describing the visual detail 

I would agree with that for a science plot.  For an example, I think the code probably stands on its own, but the visual detail should be described so the reader can match the code with the commands.